code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -494,6 +493,94 @@
 		new /obj/item/stack/sheet/metal(loc, 4)
 	qdel(src)
 
+/**
+  * # Food and Drink Cart
+  *
+  * Secure variant of the [Food Cart][/obj/machinery/smartfridge/foodcart].
+  */
+/obj/machinery/smartfridge/secure/foodcart
+	name = "E.A.T."
+	desc = "Short for the Entrée Armoured Transport, this technological marvel by Donk Co. allows Chefs to... for hawking your food and drink wares around the station, just with a bit more security."
```

Comment:
The description seems a bit off with the pause in the middle?

Reply:
Changed in latest commit